packages/react-refresh/src/__tests__/ReactFreshMultipleRenderer-test.internal.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+jest.resetModules();
+let React = require('react');
+let ReactFreshRuntime = require('react-refresh/runtime');
+ReactFreshRuntime.injectIntoGlobalHook(global);
+let ReactDOM = require('react-dom');
+
+jest.resetModules();
+let ReactART = require('react-art');
```

Comment:
These are top level because I couldn't get ART to work otherwise. Dunno why.

Reply:
Ah, that's unfortunate since it means you can't reset modules between tests.

Reply:
Yeah. I spent 20 minutes on this and then flipped the table